products/rockylinux.md

```diff
@@ -24,4 +24,11 @@ releases:
 
 > [Rocky Linux](https://rockylinux.org/) is a Linux distribution that is intended to be a downstream, complete binary-compatible release using the Red Hat Enterprise Linux (RHEL) operating system source code. The project is led by Gregory Kurtzer, founder of the CentOS project.
 
+## Differences with Upstream RHEL:
+
+ - Unlike RHEL, [Rocky Linux does not support point releases once a newer one is available.](https://forums.rockylinux.org/t/what-is-eol-of-rl8/3316/10) Once a new minor point release is available, the older one is immediately considered end of life and users must upgrade to continue receiving security updates. For example once 8.5 gets a general release, 8.4 is immediately end of life. Whereas on RHEL [this is not the case.](https://access.redhat.com/articles/rhel-eus)
+ - Does not support [live kernel patching](https://access.redhat.com/solutions/2206511), instead users who need this depend on third party paid services.
```

Comment:
is this a question ?  I  mean it will be much better if you write 
`Rocky Linux doesn't support ....` instead of a `Does not support ...`